tests/test_pipelines.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+import json
+from datetime import datetime
+from StringIO import StringIO
+import subprocess as sp
+from scrapy.exceptions import DropItem
+from pa11ycrawler.pipelines import (
+    DuplicatesPipeline, DropDRFPipeline, Pa11yPipeline
+)
+
+
+def test_duplicates_pipeline():
+    dup_pl = DuplicatesPipeline()
+    spider = object()
+    # first item: no problem
+    item1 = {"url": "google.com"}
+    processed1 = dup_pl.process_item(item1, spider)
+    assert item1 == processed1
+
+    # second item is different, so no problem
+    item2 = {"url": "edx.org"}
+    processed2 = dup_pl.process_item(item2, spider)
+    assert item2 == processed2
+
+    # third is the same as a previous, so raises an exception
+    item3 = {"url": "google.com"}
+    with pytest.raises(DropItem):
```

Comment:
a bit of a testing nit, but can you assert that, also, it does not crawl the duplicate? (depends on how the raise is handled, for example)